pictures/contrib/rest_framework.py

```diff
@@ -34,12 +35,14 @@ def to_representation(self, obj: PictureFieldFile):
             },
         }
         try:
-            query_params = self.context["request"].GET
+            query_params: QueryDict = self.context["request"].GET
         except KeyError:
             pass
         else:
             ratio = query_params.get(f"{self.source}_ratio")
             container = query_params.get(f"{self.source}_container")
+            if container is not None:
+                container = int(container)
```

Comment:
On second thought, would you mind adding a test for this line?

Reply:
I added the test here #95